spec/features/idv/flow_spec.rb

```diff
@@ -175,6 +175,20 @@
       expect(page).to have_content(t('idv.form.ccn'))
     end
 
+    scenario 'credit card field only allows numbers', js: true do
+      _user = sign_in_and_2fa_user
+
+      visit verify_session_path
+
+      fill_out_idv_form_ok
+      click_button 'Continue'
+
+      find('#idv_finance_form_finance_type_ccn', visible: false).trigger('click')
+      find('#idv_finance_form_ccn').native.send_keys('abcd1234')
+
+      expect(find('#idv_finance_form_ccn').value).to eq '1234'
```

Comment:
interesting! does this also tell the end user what the issue is? Somtimes I fill in forms and this type of re-formatting happens and it is confusing...